x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/action/TransportStartDataFrameAnalyticsAction.java

```diff
@@ -331,6 +384,19 @@ public void onTimeout(TimeValue timeout) {
         });
     }
 
+    private static class StartContext {
+        private final DataFrameAnalyticsConfig config;
+        private final List<PhaseProgress> progressOnStart;
+
+        private StartContext(DataFrameAnalyticsConfig config, List<PhaseProgress> progressOnStart) {
+            this.config = config;
+            this.progressOnStart = progressOnStart;
+            for (PhaseProgress phase : progressOnStart) {
+                LOGGER.info("[{}] Progress [{}] is [{}]", config.getId(), phase.getPhase(), phase.getProgressPercent());
```

Comment:
seems like a `debug` level to me?

Reply:
Good catch, that was a leftover. We don't really need it at all.